pkgs/development/web/nodejs/v22.nix

```diff
@@ -17,9 +17,6 @@ buildNodejs {
     ./bypass-darwin-xcrun-node16.patch
     ./node-npm-build-npm-package-logic.patch
     ./use-correct-env-in-tests.patch
-    (fetchpatch2 {
-      url = "https://github.com/nodejs/node/commit/534c122de166cb6464b489f3e6a9a544ceb1c913.patch";
```

Comment:
https://github.com/nodejs/node/commit/33242ff042200f653568a496518f9b78124ec2da is in 22.3.0